nixos/modules/misc/ids.nix

```diff
@@ -284,6 +284,7 @@
       glance = 266;
       couchpotato = 267;
       gogs = 268;
+      trezord = 269;
```

Comment:
@joachifm How do I use a non-static uid then? Do I just not give a uid/gid to `extraUsers`/`extraGroups`?